<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormVeginereChiper</class>
 <widget class="QWidget" name="FormVeginereChiper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>TUGAS 3 MATAKULIAH KRIPTOGRAFI DAN KEAMANAN SISTEM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>APLIKASI VEGINERE CIPHER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 255)</string>
   </property>
   <property name="text">
    <string>OLEH: AZHAR ABDUL SALI (C2C022103)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 170, 0)</string>
   </property>
   <property name="text">
    <string>PLAINTEXT   :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 170, 0)</string>
   </property>
   <property name="text">
    <string>KEY              :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 170, 0)</string>
   </property>
   <property name="text">
    <string>CHIPERTEXT :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 170, 0)</string>
   </property>
   <property name="text">
    <string>PLAINTEXT   :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="plainTextEdit_plain">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="plainTextEdit_plain2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="plainTextEdit_cipher">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="plainTextEdit_key">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_encrypt">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ENKRIPSI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_decrypt">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>DEKRIPSI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
